src/openrct2/drawing/Text.cpp

```diff
@@ -196,6 +196,12 @@ void gfx_draw_string_left_clipped(
     DrawTextEllipsisedCompat(dpi, x, y, width, format, args, colour, TextAlignment::LEFT);
```

Comment:
Make the old one call the new one, so we don't duplicate code.

Reply:
This issue is still pending. Are there places still calling this old function? If not, delete it, if there are, then this should be:
```suggestion
    gfx_draw_string_left_clipped(dpi, format, args, colour, { x, y }, width);
```